examples/schneider_report.py

```diff
@@ -11,10 +11,10 @@
 from squad_client.core.models import Squad
 
 
-SquadApi.configure(url='https://qa-reports.linaro.org/', token=os.getenv('QA_REPORTS_TOKEN'))
+SquadApi.configure(url='https://qa-reports.linaro.org/', token='3ea92334e43662c171fc459b79f283b44bea81b4')
```

Comment:
exposing token in the open repository isn't a good idea.

Reply:
Oops..this was forgotten during testing. I will fix this.

Reply:
We burned that token, it no longer works.